### ISSUE TITLE
mysql_query without DB connection makes crash

### ISSUE THREAD

**Issue by canasta** [facebook/hhvm#4880]
When I use mysql_query() without mysql_connect() and mysql_select_db(), it makes crash.

in /var/log/hhvm/error.log
```
Notice: Undefined variable: connect in /***/www/test.php on line 9
Core dumped: Segmentation fault
Stack trace in /tmp/stacktrace.6059.log
```

hhvm --version
HipHop VM 3.5.0 (rel)

**Comment by DmitrySoshnikov** [comment]
@canasta, we weren't able to repro it on a simple test. Do you have an isolated test?

Also, it very likely was fixed in 3.6.

**Comment by paulbiss** [comment]
Please try this with the nightly to see if it's been fixed by d6577daf89598be8bdaa42d7489950b7fd8da265

**Comment by Orvid** [comment]
This issue has been waiting for more info for more than 2 weeks. Closing for now, feel free to re-open it if you can provide more info.

https://github.com/facebook/hhvm/wiki/Human-Timeouts
